README.md

```diff
@@ -1,3 +1,19 @@
-# Walkthrough Template
+# Solution Pattern Template
 
-A template that can be used to bootstrap new walkthroughs. Walkthrough content goes under `/walkthroughs`.
+A template that can be used to bootstrap new Solution Patterns. Content goes under the `/walkthroughs` folder.
+
+## Adding Managed Services
+If you would like to include managed services with your Solution Pattern, you can add them to the walkthrough.json file under `Dependencies`.<br>
+
+Example:
```

Comment:
Should we also describe completely custom services, e.g. https://github.com/integr8ly/solution-pattern-template/blob/master/walkthroughs/1-template-walkthrough/walkthrough.json#L11 ?

Reply:
yes, that would be a good idea. I'm not sure I know enough about this though to be able to put something together. Any chance you could provide something?

Reply:
100% - we need to document which API Objects are valid 👍